examples/plugins/ebpf/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ubuntu:wily
```

Comment:
I've looked into it, but unfortunately bcc only provides binaries for Ubuntu/Fedora. bcc doesn't ship on pip (probably because of the C-library dependencies) and it's a bit of a pain to build. See https://github.com/iovisor/bcc/blob/master/INSTALL.md